Makefile

```diff
@@ -42,7 +41,6 @@ rebuild: clean
 prune: clean
 	@echo -e "${RED}Cleaning all docker environment...${RESET}"
 	@docker system prune -a -f --volumes
-	rm -rf frontend/node_modules backend/node_modules

```

Comment:
@tyrossel Why not removing node_modules on `prune` command ? We need to have a command to clear modules in order to clean up all the project